Tribler/Core/Utilities/install_dir.py

```diff
@@ -16,7 +17,7 @@ def is_frozen():
     """
     try:
         # PyInstaller creates a temp folder and stores path in _MEIPASS
-        base_path = sys._MEIPASS
+        sys._MEIPASS
```

Comment:
I guess this was assigned to a variable before as it could get optimized out if not assigned. We should compile this change in PyInstaller with optimization enabled and see what happens before removing.

Reply:
I'll just change it back.